src/Html.tsx

```diff
@@ -133,3 +133,8 @@ export const HTML = React.forwardRef(
     return <group {...props} ref={group} />
   }
 )
+
+export const HTML = (props: HtmlProps, ref: React.Ref<HTMLDivElement>) => {
```

Comment:
Should this have type signatures? If so, should it use the React.forwardRef HOF?

Reply:
type signatures? im not so good with ts, what's that? i mean, it should just behave like the original and have the same types-

Reply:
Sorry, this is my first time writing any TS! I mean, should it look more like this:
```
export const HTML = React.forwardRef(
  (props: HtmlProps, ref: React.Ref<HTMLDivElement>) => {
    console.warn('danger');
    return <Html {...props} ref={ref} />;
  }
);
```
or can I do something like this:
```
const HTML = (...args) => {
  console.warn('danger');
  return <Html {...args} />;
}
```

Reply:
The first one looks good to me, I would consider putting the warning in a useEffect(() => ..., [])

Reply:
I'm also worried `<HTML>` isn't supposed have its own `ref`, in case it's pointing to the wrong place. This is probably a non-issue, my knowledge of the React API is out-of-date.